openapi/disqus/types.bal

```diff
@@ -0,0 +1,243 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+# Category
+public type PostObject record {
```

Comment:
Seems like all the fields are nillable and optional. Is it how API is supported ?

Reply:
The API response of this API is not informatively documented with this detail. It provides the response through an example as in https://disqus.com/api/docs/posts/list/. Therefore to give the benefit of the doubt it is made nillable.